src/TLuaInterpreter.cpp

```diff
@@ -15449,6 +15450,69 @@ int TLuaInterpreter::getConnectionInfo(lua_State *L)
     return 2;
 }
 
+int TLuaInterpreter::unzipAsync(lua_State *L)
+{
+    QString zipLocation;
```

Comment:
It is purely a technical thingummy - but in fixing that anti-pattern you can now move the declaration down to where the value is assigned, i.e. so that the latter now can be something like:
```CPP
    QString zipLocation{QString::fromUtf8(lua_tostring(L, 1))};
```
all in one line...! :grinning:

Reply:
Good catch!